app/main/views.py

```diff
@@ -33,6 +33,139 @@ def index_g_cloud():
+        'crumbs': [
+            {
+                'text': 'Specialists to work on digital projects',
+                'link': 'https://digitalservicesstore.service.gov.uk'
+            },
+            {
+                'text': 'Framework information'
+            },
+        ]
+    })
+    return render_template(
+        'content/framework-digital-services.html', **template_data
+    )
+
+
+@main.route('/crown-hosting')
+def index_crown_hosting():
+    template_data = get_template_data(main, {
+        'title': 'Physical datacentre space for legacy systems – Digital Marketplace',  # noqa
+        'crumbs': [
+            {'text': 'Physical datacentre space for legacy systems'}
+        ]
+    })
+    return render_template('content/index-crown-hosting.html', **template_data)
+
+
+@main.route('/crown-hosting/framework')
+def framework_crown_hosting():
+    template_data = get_template_data(main, {
+        'title': 'Crown Hosting Data Centres framework – Digital Marketplace',
+        'crumbs': [
+            {
+                'text': 'Physical datacentre space for legacy systems',
+                'link': url_for('.index_crown_hosting')
+            },
+            {
+                'text': 'Framework information'
+            },
+        ]
+    })
+    return render_template(
+        'content/framework-crown-hosting.html', **template_data
+    )
+
+
+@main.route('/buyers-guide')
+def buyers_guide():
+    template_data = get_template_data(main, {
+        'title': 'Buyers guide – Digital Marketplace',
+        'crumbs': [
+            {
+                'text': 'Buyers\' guide'
+            }
+        ]
+    })
+    return render_template('content/buyers-guide.html', **template_data)
+
+
+@main.route('/suppliers-guide')
+def suppliers_guide():
+    return redirect('/g-cloud/suppliers-guide', 301)
+
+
+@main.route('/g-cloud/buyers-guide')
+def buyers_guide_g_cloud():
+    template_data = get_template_data(main, {
+        'title': 'G-Cloud buyers\' guide – Digital Marketplace',
+        'crumbs': [
+            {
+                'text': 'Cloud technology and support',
+                'link': url_for('.index_g_cloud')
+            },
+            {
+                'text': 'Buyers\' guide'
+            }
+        ]
+    })
+    return render_template(
+        'content/buyers-guide-g-cloud.html', **template_data
+    )
+
+
+@main.route('/g-cloud/suppliers-guide')
+def suppliers_guide_g_cloud():
+    template_data = get_template_data(main, {
+        'title': 'G-Cloud suppliers\' guide – Digital Marketplace',
+        'crumbs': [
+            {
+                'text': 'Cloud technology and support',
+                'link': url_for('.index_g_cloud')
+            },
+            {
+                'text': 'Suppliers\' guide'
+            }
+        ]
+    })
+    return render_template(
+        'content/suppliers-guide-g-cloud.html', **template_data
+    )
+
+
+@main.route('/terms-and-conditions')
```

Comment:
Updated the pull request description to mention that it also closes [story 94382950](https://www.pivotaltracker.com/story/show/94382950).